src/arrayMethodSort.js

```diff
@@ -4,8 +4,32 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
Adding methods to `Array.prototype` is generally discouraged as it can lead to conflicts with other code or future JavaScript versions. Consider using a different approach, such as a utility function, to achieve similar functionality.